src/Meshes/mesh.ts

```diff
@@ -1302,18 +1302,41 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
      * This method recomputes and sets a new BoundingInfo to the mesh unless it is locked.
      * This means the mesh underlying bounding box and sphere are recomputed.
      * @param applySkeleton defines whether to apply the skeleton before computing the bounding info
+     * @param applyMorph  defines whether to apply the morph target before computing the bounding info
      * @returns the current mesh
      */
-    public refreshBoundingInfo(applySkeleton: boolean = false): Mesh {
+    public refreshBoundingInfo(applySkeleton: boolean = false, applyMorph: boolean = false): Mesh {
         if (this._boundingInfo && this._boundingInfo.isLocked) {
             return this;
         }
 
         const bias = this.geometry ? this.geometry.boundingBias : null;
-        this._refreshBoundingInfo(this._getPositionData(applySkeleton), bias);
+        this._refreshBoundingInfo(this._getPositionData(applySkeleton, applyMorph), bias);
         return this;
     }
 
+    /** @hidden */
+    public _getPositionData(applySkeleton: boolean, applyMorph: boolean): Nullable<FloatArray> {
+        var data = super._getPositionData(applySkeleton, applyMorph);
+
+        if (data && applyMorph && this.morphTargetManager) {
+
+            for (let vertexCount = 0; vertexCount < data.length; vertexCount++) {
+
+                for (let targetCount = 0; targetCount < this.morphTargetManager.numTargets; targetCount++) {
+                    var influence = this.morphTargetManager.getTarget(targetCount).influence;
+                    if (influence > 0.0) {
+                        var morphTargetPositions = this.morphTargetManager.getTarget(targetCount).getPositions();

```

Comment:
even thou it is 0(1) operation, could you reuse the `this.morphTargetManager.getTarget(targetCount)` call? something like:

```suggestion
                        const target = this.morphTargetManager.getTarget(targetCount);
                            if(target.influence > 0) {
                                const morphTargetPositions = target .getPositions();
```